confignode/pom.xml

```diff
@@ -40,16 +40,6 @@
             <artifactId>service-rpc</artifactId>
             <version>${project.version}</version>
         </dependency>
-        <dependency>
-            <groupId>org.apache.iotdb</groupId>
-            <artifactId>iotdb-thrift-confignode</artifactId>
-            <version>${project.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.iotdb</groupId>
-            <artifactId>node-commons</artifactId>
```

Comment:
why remove iotdb-thrift-confignode and node-commons?

Reply:
We clean the maven dependency. See [this doc](https://y8dp9fjm8f.feishu.cn/docs/doccneJxfaW4XZ7igbr7VrIo2bf#) for more details.